v4-client-py-v2/dydx_v4_client/node/client.py

```diff
@@ -443,6 +454,166 @@ async def get_authenticators(
             accountplus_query.GetAuthenticatorsRequest(account=address)
         )
 
+    async def get_node_info(self) -> tendermint_query.GetNodeInfoResponse:
+        """
+        Query for node info.
+
+        Returns:
+            tendermint_query.GetNodeInfoResponse: The response containing the node information.
+        """
+        return tendermint_query_grpc.ServiceStub(self.channel).GetNodeInfo(
+            tendermint_query.GetNodeInfoRequest()
+        )
+
+    async def get_delegation_total_rewards(
+        self, address: str
+    ) -> distribution_query.QueryDelegationTotalRewardsResponse:
+        """
+        Get all unbonding delegations from a delegator.
+
+        Args:
+            address (str): The delegator address
+
+        Returns:
+            distribution_query.QueryDelegationTotalRewardsResponse: All unbonding delegations from a delegator.
+        """
+        return distribution_query_grpc.QueryStub(self.channel).DelegationTotalRewards(
+            distribution_query.QueryDelegationTotalRewardsRequest(
+                delegator_address=address
+            )
+        )
+
+    async def get_all_gov_proposals(
+        self,
+        proposal_status: Optional[str] = None,
+        voter: Optional[str] = None,
+        depositor: Optional[str] = None,
```

Comment:
These arguments are non-optional in rust implementation, I feel like they shouldn't be optional, because Rust bindings are generated automatically from specification